src/continuous_aggregate/migrations/0002_deploy_v1_continuous_aggregate.py

```diff
@@ -0,0 +1,983 @@
+from django.db import migrations
+
+from continuous_aggregate.view_definitions import VIEW_STRINGS, get_view_strings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('continuous_aggregate', '0001_initial'),
+    ]
+
+    _LOG_SCHEMA_NAME        = "log"
+    _LOG_TABLE_NAME         = "execution_log"
+    _LOG_FUNCTION_NAME      = "proc_execution_log_entry"
+
+    _PROCESS_SCHEMA_NAME    = "prc"
+    _PROCESS_FUNCTION_NAME  = "proc_pre_post_process"
+
+
+    _sql_log_schema              = f"CREATE SCHEMA IF NOT EXISTS {_LOG_SCHEMA_NAME}"
```

Comment:
Gebruik je nu toch aparte schema's? We zouden alles in het public schema doen toch?